frontend/src/pages/Workflows/WorkflowContainer.js

```diff
@@ -96,26 +102,36 @@ class WorkflowContainer extends Component {
             subprojectId={this.subprojectId}
             canViewPermissions={canViewPermissions}
             canAssignSubproject={canAssignSubproject}
-            closeSubproject={this.closeSubproject}
             canCloseSubproject={canCloseSubproject}
           />
 
           {this.props.permissionDialogShown ? (
             <WorkflowItemPermissionsContainer projectId={this.projectId} subProjectId={this.subprojectId} />
           ) : null}
-          <Workflow
-            {...this.props}
-            projectId={this.projectId}
-            subProjectId={this.subprojectId}
-            closeWorkflowItem={this.closeWorkflowItem}
-          />
+          <Workflow {...this.props} projectId={this.projectId} subProjectId={this.subprojectId} />
           <WorkflowDialogContainer location={this.props.location} />
           <AdditionalInfo
             resources={this.props.workflowItems}
             isAdditionalDataShown={this.props.isWorkflowitemAdditionalDataShown}
             hideAdditionalData={this.props.hideWorkflowitemAdditionalData}
             {...this.props}
           />
+          <DialogForClose
```

Comment:
Since the behavior/props can be passed to DialogForClose component why using 2? If you really want to split them create own components ProjectDialogForClose/SubprojectDialogForClose/WorkflowitemDialogForClose.